app/packages/prometheus/src/components/Chart.tsx

```diff
@@ -0,0 +1,182 @@
+            <Square sx={{ color: datum.customColor }} />
+            <Box
+              component="span"
+              sx={{ flexGrow: 1, overflow: 'hidden', textOverflow: 'ellipsis', whiteSpace: 'nowrap' }}
+            >
+              {datum.customLabel}
+            </Box>
+            <Box component="span" sx={{ textAlign: 'right', whiteSpace: 'nowrap' }}>
+              {/* eslint-disable-next-line @typescript-eslint/no-explicit-any */}
+              {roundNumber(yValue, 4)} {(props as any).text}
+            </Box>
+          </Box>
+        </Box>
+      </foreignObject>
+    </g>
+  );
+};
+
+/**
+ * `IChartProps` is the interface for the properties of the `Chart` component.
+ */
+interface IChartProps {
+  max?: number;
+  metrics: IMetric[];
+  min?: number;
+  setTimes?: (times: ITimes) => void;
+  stacked: boolean;
+  times: ITimes;
+  type: 'line' | 'area' | 'bar';
+  unit?: string;
+}
+
+/**
+ * The `Chart` component is used to render a line, area or bar chart for the provided `metrics`. The chart type is
+ * specified via the `type` property. A user can also stack the metrics by setting the `stacked` property. The `min`,
+ * `max` and `times` are required, to set the domain of the chart.
+ *
+ * The chart can also be used to adjust the currently selected `times`, by selecting a time range in the chart, which
+ * will then execute the provided `setTimes` function.
+ */
+const Chart: FunctionComponent<IChartProps> = ({ metrics, type, stacked, unit, min, max, times, setTimes }) => {
+  const theme = useTheme();
+  const refChart = useRef<HTMLDivElement>(null);
+  const chartSize = useDimensions(refChart);
+
+  /**
+   * Depending on the provided `type` we have to use different components for the chart. This is handled in the
+   * following.
+   */
+  const chartData = metrics.map((metric, index) =>
+    type === 'area' ? (
+      <VictoryArea
+        key={metrics[index].label}
+        data={metric.data}
+        name={metrics[index].label}
+        colorScale={[metrics[index].color]}
+        interpolation="monotoneX"
+      />
+    ) : type === 'bar' ? (
+      <VictoryBar
+        key={metrics[index].label}
+        data={metric.data}
+        name={metrics[index].label}
+        colorScale={[metrics[index].color]}
+      />
+    ) : (
+      <VictoryLine
+        key={metrics[index].label}
+        data={metric.data}
+        name={metrics[index].label}
+        colorScale={[metrics[index].color]}
+        interpolation="monotoneX"
+      />
+    ),
+  );
+
+  /**
+   * The `BrushVoronoiContainer` component is used as container for the charts. It allows us to render a tooltip for the
+   * metrics and to select a new time range via the brush function of the Victory charts package.
+   */
+  const BrushVoronoiContainer = createContainer<VictoryVoronoiContainerProps, VictoryBrushContainerProps>(
+    'voronoi',
+    'brush',
+  );
+
+  return (
+    <Box height="100%" width="100%" ref={refChart}>
+      <VictoryChart
```

Comment:
The y-axis range could use some padding, when two queries are used. Currently the y-max is set to the maximum value of the result data (same with the y-min). This is especially problematic when the result doesn't change, see screenshot:
![image](https://user-images.githubusercontent.com/60616628/224675009-dade9fd9-9140-4dad-89bd-6f71641c84fb.png)

Or use this search query: `queries[]=grafana_api_dashboard_get_milliseconds_count&queries[]=grafana_access_evaluation_count`